_data/events_gdcr2019/shenzhen.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "GDCR19 in Shenzhen",
+  "url": "",
```

Comment:
Hi @bogdan-tkach! Great! Congrats for it! Thanks for including your event! Please, we'd need an `url` for the event. It can be a Meetup page, a webpage about the event, an Eventbrite page, ... Any page where an attendee can find the information about your event and know how to register. You can see examples on the included events until now: https://www.coderetreat.org/events/ When you have the url of that page, please edit the file to include it and push it again in the same branch. Don't worry. We'll keep this pull request open until you have that `url`. Thanks!